open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/server/AnalyticsModelingRestServices.java

```diff
@@ -225,6 +226,39 @@ public AnalyticsModelingOMASAPIResponse createArtifact(String serverName, String
 		return ret;
 	}
 
+	/**
+	 * Update analytics artifact defined as json input.
+	 * @param serverName where to create artifact.
+	 * @param userId requested the operation.
+	 * @param serverCapability source where artifact persist.
+	 * @param artifact definition.
+	 * @return response with artifact or error description.
+	 */
+	public AnalyticsModelingOMASAPIResponse updateArtifact(String serverName, String userId, String serverCapability, AnalyticsAsset artifact) {
```

Comment:
It may be error response org.odpi.openmetadata.accessservices.analyticsmodeling.responses.ErrorResponse.ErrorResponse(AnalyticsModelingCheckedException)